### PR TITLE
docs: add build-doc.sh

### PR DESCRIPTION
**Pull request by dingbo8128**

### CODE REVIEW

File: ci/build-doc.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+pip3 install pdoc3
+source .env
+pdoc --html --force --output-dir ../docs taosrest
+pdoc --html --force --output-dir ../docs taos
+# copy generated html files to Document Server
+sshpass -p "$DOC_HOST_PWD"  scp -r ../docs/* root@$DOC_HOST:/data/apidocs/taospy/
```
Comment (zitsen) [inline ci/build-doc.sh:7]: just use PW or PASSWD or PASSWORD, not PWD

File: ci/build-doc.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+pip3 install pdoc3
+source .env
```
Comment (zitsen) [inline ci/build-doc.sh:3]: use `[ -e .env ] && source .env` to make it pass when use command environment variables.

File: pyproject.toml [hunk 3]
```diff
@@ -30,7 +30,6 @@ pytest = [
   { version = "^4.6", python = ">=2.7,<3.0" },
   { version = "^6.2", python = ">=3.7,<4.0" }
 ]
-pdoc = { version = "^7.1.1", python = "^3.7" }
```
Comment (zitsen) [inline pyproject.toml:33]: Removing an entry from pyproject.toml requires `poetry update`

**Review by zitsen — CHANGES_REQUESTED** [review]
See my comments

**Review by zitsen — APPROVED** [review]
LGTM.

**Review by sangshuduo — COMMENTED** [review]
LGTM

**Pull request merged**